angular-frontend/accelerator.yaml

```diff
@@ -0,0 +1,42 @@
+accelerator:
+  displayName: Angular Frontend
+  description: Angular Single Page Application with client side rendered web UI. Application provides Service to interact with backend API's.
+  tags:
+    - angular
+    - web
+    - tanzu
+
+  options:
+    - name: artifactId
+      inputType: text
+      defaultValue: "client-side-ui-starter"
+      label: Application artifact name
+      required: true
+    - name: backendService
+      inputType: text
+      defaultValue: "customer-profile-backend.namespace.svc.cluster.local"
+      label: Hostname of the backend services app. If backend services are deployed to the same cluster then a local route can be used (e.g. <backend-app>.<namespace>.svc.cluster.local).
```

Comment:
Do you need the entire svc.cluster.local domain suffix for local cluster deployments?  Would it be easier to just use <backend-app>.<namespace>?

Reply:
I think we tried '<backend-app>.<namespace>' without the suffix and it didn't work. I'll check it again to be sure.

Reply:
It works! Thanks for the hint!